rust/agama-lib/src/base_http_client.rs

```diff
@@ -28,6 +28,15 @@ pub struct BaseHTTPClient {
 const API_URL: &str = "http://localhost/api";
 
 impl BaseHTTPClient {
+    // Provides new client with no specialities like authorization headers or so.
+    // If needed, there is place for helpers like "self::to_authorized" or so.
+    pub fn bare() -> Result<Self, ServiceError> {
```

Comment:
Please consider using https://github.com/openSUSE/agama/pull/1517 instead: `BaseHTTPClient::default()` makes the same thing as your `bare()`

Reply:
i'll fix it once your pr is merged